pkg/registry/servicecatalog/instance/storage.go

```diff
@@ -0,0 +1,44 @@
+package instance
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/rest"
+	genericregistry "k8s.io/kubernetes/pkg/registry/generic"
+	"k8s.io/kubernetes/pkg/runtime"
+)
+
+type serviceInstanceStorage struct {
+}
+
+// NewStorage creates a new rest.Storage responsible for accessing Instance
+// resources
+func NewStorage() rest.Storage {
```

Comment:
We do need one for each type, but `genericregistry.Store` takes a lot of the pain of writing them away.